src/Order/OrdersList.php

```diff
@@ -107,12 +107,12 @@ public function add_order_barcode_column_content( $column, $order_id ) {
 	 * @return array.
 	 */
 	public function add_order_delivery_date_column_header( $columns ) {
-		$wc_actions = $columns['wc_actions'];
-		unset( $columns['wc_actions'] );
-
 		$columns['postnl_delivery_date'] = esc_html__( 'Delivery Date', 'postnl-for-woocommerce' );
-		$columns['wc_actions']           = $wc_actions;
-
+		if( isset( $columns['wc_actions'] ) ){
+			$wc_actions = $columns['wc_actions'];
+			unset( $columns['wc_actions'] );
+			$columns['wc_actions']       = $wc_actions;
+		}
```

Comment:
We are unsetting the **'wc_actions'** to add `postnl_delivery_date` before it, so this code will do nothing actually its just unset then set it.